server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/Main.scala

```diff
@@ -32,12 +32,8 @@ object Main extends IOApp {
         blockingEc,
         metadata => List(metadata.localPath -> metadata)
       )
-      googleStorageService <- Stream.resource(GoogleStorageService.fromApplicationDefault())
```

Comment:
^ `implicit0(it: Linebacker[IO]) <-` can probably be removed here right, since it's just needed for `GoogleStorageService`?

Reply:
updated